### PR TITLE
Added addHeaderFromArray method to the SimpleWriter

### PR DESCRIPTION
**Pull request by bedus-creation**
Currently, no feature to directly add headers from the array. This PR adds `addHeaderFromArray` method to the SimpleWriter, which allows to add headers from array.

### CODE REVIEW

**Comment by freekmurze** [comment]
Thank you!

Could you remove all unrelated code style changes in this PR so I can easily see the actual changes?

**Comment by bedus-creation** [comment]
@freekmurze Thanks for the quick response, I have updated the PR by removing unnecessary changes.

File: src/SimpleExcelWriter.php [hunk 1]
```diff
@@ -127,6 +127,18 @@ public function addRows(iterable $rows, Style $style = null)
         return $this;
     }
 
+    public function addHeaderFromArray(array $header): self
```
Comment (freekmurze) [inline src/SimpleExcelWriter.php:130]: Rename this to `addHeader`

**Review by freekmurze — CHANGES_REQUESTED** [review]
Could you also update the readme with an example of how to use this new function?

**Comment by bedus-creation** [comment]
Readme updated.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**